app/concerns/liquid_interpolatable.rb

```diff
@@ -189,6 +189,11 @@ def uri_expand(url, limit = 5)
       url
     end
 
+    # Rebase URIs contained in attributes in a given HTML fragment
+    def rebase_html(input, base_uri)
```

Comment:
At first, I named it `html_resolve_uris`, but I didn't like it because it was lengthy and the word `uris` didn't looked nice to me.